res/translations/source_copy_allow_list.tsv

```diff
@@ -0,0 +1,436 @@
+*	14-bit (LSB)
+*	14-bit (MSB)
+*	Bitcrusher
+*	Ping Pong
+*	Triplets
+*	Q
+*	Q 1
+*	Q 2
+*	Flanger
+*	Regen
+*	Mix
+*	Autopan
+*	Bessel4 ISO
+ca,de,el,en_CA,en_GB,es,es_419,es_AR,es_CO,es_CR,es_EC,es_ES,es_MX,es_PA,es_UY,fr_CA,fr_CI,hu,it,nl,pt,pt_BR,pt_PT,sl,sv	Res
+*	%1:%2
+*	Bessel8 ISO
+*	LR8 ISO
+*	BQ EQ
+*	BQ EQ/ISO
+*	Loudness
+*	16 bits
+*	24 bits
+*	Param EQ
+*	CBR
+*	Rekordbox
+*	%1: %2
+*	F12
+*	Esc
+*	Stereo
+*	2x MSAA
+*	4x MSAA
+*	8x MSAA
+*	16x MSAA
+*	 px
+*	TextLabel
+*	&OK
+*	LPF
+*	HPF
+*	Stem #%1
+*	FPS: n/a
+*	<i>FPS: %0/%1</i>
+de,hu,bs,cs,da,es_419,es_AR,es_CO,es_CR,es_EC,es_MX,es_PA,es_UY,et,fr,hr,id,it,lb,ms,nb,nl,nn,oc,pl,pt,pt_PT,ro,sk,sl,sv,vi	Album
+de,da,fr,lb,ms,nl,oc,sv	Genre
+de,it,pl	Bitrate:
+es,es_419,es_AR,es_CO,es_MX,ca,es_ES	Error:
+pt_BR,de,el,es_419,es_AR,es_CO,es_CR,es_EC,es_MX,es_PA,es_UY,id,it,nl,ro	Deck %1
+hu,da,de,el,es_419,es_AR,es_CO,es_CR,es_EC,es_MX,es_PA,es_UY,eu,id,nb,nl,pl,pt,pt_BR,pt_PT,ro,sn,tr,vi	Sampler %1
+hu,id,nl,sn	Auxiliary %1
+*	Crossfader
+*	Cue
+hu,ca,de,et,fr,nb,pl,ro,sv,tr	Port
+hu,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh,zh_CN,zh_HK,zh_TW	A
+hu,pt,pt_PT,et,fi,nl,pl,pt_BR,ro,zh,zh_CN,zh_HK,zh_TW	Bb
+hu,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh,zh_CN,zh_HK,zh_TW	B
+hu,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh,zh_CN,zh_HK,zh_TW	C
+hu,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK,zh_TW	Db
+hu,pt,pt_PT,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh,zh_CN,zh_HK,zh_TW	D
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK,zh_TW	Eb
+hu,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh,zh_CN,zh_HK,zh_TW	E
+hu,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh,zh_CN,zh_HK,zh_TW	F
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh_CN	F#
+hu,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh,zh_CN,zh_HK,zh_TW	G
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK,zh_TW	Ab
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK	Am
+hu,et,fi,nl,pl,pt_BR,ro,zh,zh_CN,zh_HK,zh_TW	Bbm
+hu,et,fi,nl,pl,pt_BR,ro,zh,zh_CN,zh_HK,zh_TW	Bm
+hu,et,fi,nl,pl,pt_BR,ro,sl,vi,zh,zh_CN,zh_HK	Cm
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh_CN	C#m
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK,zh_TW	Dm
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK	Ebm
+hu,de,et,fi,nl,pl,pt_BR,ro,sl,vi,zh,zh_CN,zh_HK,zh_TW	Em
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh_CN	Fm
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK	F#m
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK	Gm
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK	G#m
+*	10Hz
+*	10ms
+*	5ms
+*	100Hz
+*	250ms
+*	50ms
+hu,bs,de,eo,eu,fr,hr,it,lb,lt,lv,nb,nl,pt_PT,ro,sk,sv,vi	Min
+hu,bs,de,eu,fr,hr,it,lb,nl,ro,sk,sv,vi	Max
+pt,pt_BR,ca,de,fr,it,nl,pl,pt_PT,sl,sv	Tremolo
+pt_BR,da,de,id,it,nl,pt,ro	Mixer
+pt_BR,de,it,nl,pt_PT	Reloop
+de,es,es_419,es_AR,es_CO,es_ES,es_MX,nl,pt_BR,vi	Hotcues
+de,es_CR,es_EC,es_PA,es_UY,it,nl,pt_BR,pt_PT,vi,pl	Hotcues %1-%2
+nl,pl	Hotcue index
+de,it,nl,pl	Hotcue #%1
+fr,de,sv	Information
+fr,es,es_419,es_AR,es_CO,es_CR,es_EC,es_ES,es_MX,es_PA,es_UY	Simple
+ca,es,es_419,es_AR,es_CO,es_CR,es_EC,es_ES,es_MX,es_PA,es_UY	Color
+nl	is
+nl	Cover
+nl	Track BPM: 
+nl	Artist + Title
```

Comment:
Isn't "Titel" the durch word?

Reply:
Probably yes. I will remove the translation.